### ISSUE TITLE
Add "quit" in actions

### ISSUE THREAD

**Issue by sanjayankur31** [ironthree/fedora-update-feedback#2]
I was giving karma to updates and turned out there were lots of them. At some point I wanted to stop and then continue later, but I wasn't sure if hitting ctrl C would throw away all the karma I'd already given, or if they  were submitted and I could now continue later. So maybe a "quit" action would be a good way of indicating to a user that they can stop whenever and then continue afterwords?

fedora-update-feedback-1.0.4-1.fc35.x86_64

**Comment by decathorpe** [comment]
Hitting Ctrl-C would do the right thing, as comments are submitted immediately after confirming their text/karma.
But adding a "[q]uit" action sounds like a good idea regardless. I'm already working on some improvements to f-u-f (for example, making it faster by caching bodhi login cookies instead of logging in every time), and I'll include that with the next version :)

**Comment by decathorpe** [comment]
I've implemented an "abort" action with this commit: https://github.com/ironthree/fedora-update-feedback/commit/89d17ddf6307e34ebd76eb3ec39f525b35ea45ff

I named it "abort" because "quit" would not be what it does in all cases (the checks for unpushed and obsoleted updates run *after* asking for feedback for installed updates that are in "testing" state.

It looks like this now:

```
Action ([S]kip / [i]gnore / [c]omment / [a]bort): a
Aborting.

(end of program, assuming obsoleted and unpushed checks are disabled)
```

What do you think?

**Comment by sanjayankur31** [comment]
Yeh, that works very well. :rocket:

**Comment by decathorpe** [comment]
Released with version 1.1.0, will end up in Fedora repositories soon.
